src/raftstore/store/store.rs

```diff
@@ -445,20 +445,23 @@ impl<T, C> Store<T, C> {
         // Poll all snapshot messages and handle them.
         loop {
             match self.significant_msg_receiver.try_recv() {
-                Ok(SignificantMsg::SnapshotStatus {
+                Ok(Msg::SnapshotStatus {
                     region_id,
                     to_peer_id,
                     status,
                 }) => {
                     // Report snapshot status to the corresponding peer.
                     self.report_snapshot_status(region_id, to_peer_id, status);
                 }
-                Ok(SignificantMsg::Unreachable {
+                Ok(Msg::Unreachable {
                     region_id,
                     to_peer_id,
                 }) => if let Some(peer) = self.region_peers.get_mut(&region_id) {
                     peer.raft_group.report_unreachable(to_peer_id);
                 },
+                Ok(msg) => {
```

Comment:
I prefer separating `SignificantMsg` from `Msg` giving that they doesn't share sender/receiver.